src/Avalonia.Themes.Fluent/CaptionButtons.xaml

```diff
@@ -0,0 +1,70 @@
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Style Selector="CaptionButtons">
+    <Setter Property="Foreground" Value="{DynamicResource ThemeForegroundBrush}"/>
+    <Setter Property="Canvas.Right" Value="0" />
```

Comment:
Is this necessary? Where does CaptionButtons appear in a canvas?

Reply:
done.